models/workitem_repository.go

```diff
@@ -132,6 +134,22 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, typeID string, fiel
 		Type:   typeID,
 		Fields: Fields{},
 	}
+	publicKey, err := token.ParsePublicKey(token.RSAPublicKey)
```

Comment:
I tried to move it inside controller's create method but facing problem where to put UUID (inside context maybe) so that repositories can access or pass by param.